camera_system/color_changer.py

```diff
@@ -0,0 +1,54 @@
+"""
+color_changer.py
+カメラから取得した画像を6色画像に変換する
+@author kodama0720
+"""
+
+import cv2 as cv
+import numpy as np
+
+
+class ColorChanger:
+    """画像の色変換クラス"""
+
+    __BLACK = [0, 0, 0]
+    # RGB値（赤、黄色、緑、青、白）
+    __BGR_COLOR = np.array([[0, 0, 255], [0, 255, 255], [0, 255, 0],
+                           [255, 0, 0], [255, 255, 255]])
+    # HSVの閾値（赤1、赤2、黄色、緑、青、白）
+    __LOWER = np.array([[0, 90, 0], [151, 90, 0], [16, 130, 0],
+                        [41, 70, 0], [104, 100, 0], [0, 0, 128]])
+    __UPPER = np.array([[15, 255, 255], [179, 255, 255], [40, 255, 255],
+                        [103, 255, 255], [150, 255, 255], [179, 70, 255]])
+
+    def change_color(self, path="course.png"):
+        """画像を6色画像に変換する関数"""
```

Comment:
以下のように、Type Hintsを導入すると保守性が上がるかもしれません。
また、後述するDocstringの自動生成ツールが補完してくれる情報が増えます。

関数に関する説明ですが、引数や戻り値（None以外の場合）の説明も記述してもらえると助かります。
私は、Docstringのスタイルは[Google Style](https://qiita.com/11ohina017/items/118b3b42b612e527dc1d)を採用したいと考えています（異論は歓迎します）。
理由としては、私が使用しているVS CodeのDocstring生成?ツールがデフォルトでGoogle Style（多分）なためです。

```suggestion
def change_color2(self, path: str = "course.png") -> None:
        """画像を6色画像に変換する関数.

        Args:
            path (str, optional): ファイル名. Defaults to "course.png".
        """
```

Docstring生成ツール（VS Code拡張機能）: https://qiita.com/11ohina017/items/118b3b42b612e527dc1d
Type Hintsに関する説明: https://qiita.com/papi_tokei/items/2a309d313bc6fc5661c3